### ISSUE TITLE
Section 508 Compliance 

### ISSUE THREAD

**Issue by PriyaDoIT** [Chicago/opengrid#264]
OG should be section 508 compliant.  If a user is unable to use a mouse to navigate OG, we should support keyboard accessibility.  Currently, when a user tabs through OG they aren't able to select datasets, they are only able to select Commonly Used Queries.  Once a data set is added, users can tab to the grid view.
